src/enrollments/EnrollmentCollectionEntity.js

```diff
@@ -2,6 +2,7 @@
 
 import { Entity } from '../es6/Entity.js';
 import { Rels } from '../hypermedia-constants';
+import { Actions } from 'd2l-hypermedia-constants';
```

Comment:
Oh sorry, wasn't very clear.  I meant from the `hypermedia-constants` file in this repo, just changing the line above to `import { Actions, Rels } from '../hypermedia-constants';`

Reply:
Oh okay! I didn't even see that.